dcc-portal-ui/Gruntfile.js

```diff
@@ -483,16 +483,33 @@ module.exports = function (grunt) {
           ignorePath: 'app'
         },
         files: [{
-          expand: true,
-          cwd: '<%= yeoman.app %>/develop/scripts',
-          src: ['*.js']
-        }]
+            expand: true,
+            cwd: '<%= yeoman.app %>/develop/scripts',
+            src: ['*.js']
+          },
+          {
+            expand: true,
+            cwd: '<%= yeoman.app %>/branding/scripts',
+            src: ['*.js']
+          }
+        ]
       }
     }
   });
 
   grunt.registerTask('bower-install', ['bower-install-simple']);
 
+  // A very basic task to read from process.env
+  grunt.registerTask('branding_customize', 'Override UI images and strings. see app/branding/scripts/config.js', function() {
+    if (process.env.ICGC_BRANDING) {
+      var _fileName = grunt.template.process('<%= yeoman.app %>/branding/scripts/custom.js');
+      grunt.file.write(_fileName, '$ICGC_BRANDING = ' + process.env.ICGC_BRANDING);
```

Comment:
We should have branding template in the case that we use a file.  Something like /dcc-portal-ui/app/branding/scripts/custom.js.example, so long as the example file doesn't get deployed with the app.